.gitignore

```diff
@@ -336,3 +336,7 @@ ASALocalRun/
 /_output*/
 /_output
 .DS_Store
+
+# helm chart dependencies
+charts/csi-secrets-store-provider-azure/charts/
+manifest_staging/charts/csi-secrets-store-provider-azure/charts/ 
```

Comment:
Wouldn't this impact us from making updates to the yamls in manifest_staging in the future?

Reply:
The charts directory nested inside the parent charts dir is for dependencies. We never store the driver package in the repo.

Reply:
Got it!